webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -16,6 +16,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
```

Comment:
Seems like this is not needed (anymore)?

Reply:
When I remove it things break, its used at L77

Reply:
Line 77 is a comment. You are looking at the wrong file I think

Reply:
You think correctly.